.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+- "node"
+- "4.2"
+before_install:
+- rvm install 2.2.3
```

Comment:
Are you pinned to this specific version of Ruby for a reason, or could you just do `rvm install stable` instead?